Guide review. (#2)
hazelcast-guides/springboot-tomcat-session-replication: doc/modules/ROOT/pages/index.adoc

Imagine that you have a microservice which uses Tomcat HTTP sessions, and you would like to scale it horizontally. You can create multiple microservice instances, and use a load balancer to access them. However, each microservice instance will not have the same session data and the session won't be consistent through microservices when the requests reach different instances. One option is to use sticky sessions but even if you use sticky sessions, you will lose some session data when a microservice instance is crashed/shut down. To prevent any data loss and provide consistency, session data should be replicated through microservice instances.

> Multiple solutions exist for different environments and setups but in this blog post, we will find out how we can replicate sessions through Spring Boot microservices using Hazelcast with only minimal configuration settings. We will also show how it can be run on Kubernetes through a simple demo application.
> 

Reviewer: ```suggestion
Multiple solutions exist for different environments and setups but in this blog post, we will find out how we can replicate sessions through Spring Boot microservices using Hazelcast with only minimal configuration settings.

```

Author: :hushed:

> There are multiple solutions exist for different environments and setups but in this blog post, we will find out how we can replicate sessions through Spring Boot microservices using Hazelcast with only minimal configuration settings. We will also show how it can be run on Kubernetes environment through a simple demo application.

Reviewer 2: ```suggestion
Multiple solutions exist for different environments and setups but in this blog post, we will find out how we can replicate sessions through Spring Boot microservices using Hazelcast with only minimal configuration settings. We will also show how it can be run on Kubernetes through a simple demo application.
```

Open another terminal session and start the second application:
[source,bash]
----
$ java -Dserver.port=8081 -jar target/springboot-tomcat-session-replication-0.1.0.jar
----

> Now, you can make calls to application endpoints to put data and get it back. Open another terminal session and send a request to `/put` endpoint. This will set the key-value pair as a session attribute. Since session data is stored in Hazelcast distributed map, we can read this attribute from the other application instance using the same cookie:

Reviewer 2: ```suggestion
Now, you can issue HTTP calls to application endpoints to put data and get it back. Open another terminal session and send a request to `/put` endpoint. This will set the key-value pair as a session attribute. 

Since session data is stored in Hazelcast distributed map, we can read this attribute from the other application instance using the same cookie:
```

_make calls_ sounds like a suboptimal choice of words

Open another terminal session and start the second application:
[source,bash]
----

> $ java -Dserver.port=8081 -jar target/springboot-tomcat-session-replication-0.1.0.jar

Reviewer: nit: I'd avoid adding `$` if it's a one-liner command. Because it's more difficult to copy and execute in your environment.

Author: You are right, removed all.
